rel/config.exs

```diff
@@ -0,0 +1,56 @@
+# Import all plugins from `rel/plugins`
```

Comment:
This entire file (```rel/config.exs```) is generated by Distillery as a template. The template is good enough to get a release built on its own, although this file is meant to be customized.

For example, you'll most likely want to change how you generate and access your Erlang Magic Cookie below, and probably not commit it into source control!